hybridCarboniteMetadataService/src/main/scala/cromwell/services/metadata/hybridcarbonite/CarbonitedMetadataThawingActor.scala

```diff
@@ -21,15 +21,15 @@ final class CarbonitedMetadataThawingActor(carboniterConfig: HybridCarboniteConf
   startWith(CarbonitedMetadataThawingActorPendingState, CarbonitedMetadataThawingActorPendingData)
 
   when(CarbonitedMetadataThawingActorPendingState) {
-    case Event(ThawCarbonitedMetadata(workflowId), CarbonitedMetadataThawingActorPendingData) =>
+    case Event(ThawCarbonitedMetadata(rootWorkflowId), CarbonitedMetadataThawingActorPendingData) =>
 
       val response = for {
-        rawResponseFromIoActor <- asyncIo.contentAsStringAsync(carboniterConfig.makePath(workflowId), maxBytes = Option(30 * 1024 * 1024), failOnOverflow = true)
+        rawResponseFromIoActor <- asyncIo.contentAsStringAsync(carboniterConfig.makePath(rootWorkflowId), maxBytes = Option(30 * 1024 * 1024), failOnOverflow = true)
         parsedResponse <- Future.fromTry(parse(rawResponseFromIoActor).toTry.map(GcsMetadataResponse.apply))
       } yield parsedResponse
 
       response pipeTo self
-      serviceRegistry ! GetLabels(workflowId)
+      serviceRegistry ! GetRootAndSubworkflowLabels(rootWorkflowId)
```

Comment:
Reminder to update the `HybridMetadataServiceActor` to forward this message to the correct (classic) sub-actor

Reply:
will do in the next PR